### DISCUSSION TITLE
Enable nuxt i8n translation in few pages

### DISCUSSION THREAD

**Discussion by rahulkumarsingh73690** [nuxt-modules/i18n#1362] [Q&A]
Hi,

I have a nuxt.js site with more than 150+ pages and I want to translate only 10 pages in different languages.

Can you please guide me?

I don't want to place this on nuxtI18n: false on more than 140 pages. Please suggest anything better.

**Comment by rchl** [comment]
Don't think that's possible right now.

PRs welcome.

**Comment by eMeRiKa13** [comment]
> Hi,
> 
> I have a nuxt.js site with more than 150+ pages and I want to translate only 10 pages in different languages.
> 
> Can you please guide me?
> 
> I don't want to place this on nuxtI18n: false on more than 140 pages. Please suggest anything better.

Could you not create a plugin, put false by default except for your 10 routes where you want the translation?

**Reply by rahulkumarsingh73690** [reply]
If it is possible please provide more details

**Reply by eMeRiKa13** [reply]
It's a nuxt3 example for route guard. You may create a similar plugin who checks your routes and enables or not the translations

https://github.com/nuxt/framework/discussions/2701
